### ISSUE TITLE
Make Lists easier to access

### ISSUE THREAD

**Issue by ShredderWalnut** [metabolist/metatext#69]
Hi, congrats on the new 1.5.1 update! 

# Rationale

Many fedi users including me follow a huge number of accounts and split them into a handful of lists. Then we only browse those lists and no longer use the Home timeline at all.

Currently, it requires multiple steps to access lists:

1. Click my avatar at the top left corner
2. Click "Lists"
3. Click on the specific list

And if I'm done browsing one list and want to browse another, I have to swipe left to exit the list timeline and start all over again from step 1.

# Request

My feature request is to make it easier to access and switch between lists. I don't have any concrete proposals for how to adapt the UI.

Thank you for this great app!

**Comment by kerim** [comment]
Came here to make this request.

**Comment by Jmuccigr** [comment]
Yes, there should be a toolbar icon for them at the bottom, perhaps.

Also an option for accounts in lists not to appear in the home timeline at all.

mastodon seems to be different from the bird app where you can put an account in a list without also following them.

**Comment by dgoosens** [comment]
+1 for this

**Comment by rpkoller** [comment]
I would suggest to go with a pattern for example Tweetbot applies: 
![IMG_1B1A19B05ADF-1](https://user-images.githubusercontent.com/1665422/200304243-63a3689b-59bd-431a-b243-a599a08f2cee.jpeg)
you have the active list or timeline shown in the toolbar on top (accessibility in that case). and if you keep the toolbar title pressed the list of available timelines and lists for your account is shown. that way you are able to switch between lists and timelines easily.

**Comment by alper** [comment]
I don't see space really to tap the title bar the way it is now.

![CleanShot 2022-11-11 at 18 01 17@2x](https://user-images.githubusercontent.com/72429/201391757-73379c78-2264-429a-9b83-fa430a92850f.png)

**Comment by alper** [comment]
I think a long tap on Timelines is maybe a better way to access it (also a Tweetbot pattern).

**Comment by rpkoller** [comment]
probably i've used the wrong terminology and toolbar was sort of misleading. apologies. i simply meant if you press the active list label `accessibility` in my screenshot then the list of available lists shows up. in the screenshot you've provided for metatext @alper a tabbed nav with three options `home` `local` and `federated` is used. that tabbed nav could be replaced with the pattern known from tweetbot. and the list of options shown on pressed tab could contain aside the available lists, the options `home`, `local`, and `federated` on top of the list.

**Comment by kerim** [comment]
Another UI I've seen is a swipe gesture to switch between lists. Bad for discoverability, but once you know how to use it, fairly easy.

**Comment by paulp74** [comment]
This is about the only thing that I feel tooot does better than Metatext. I like the way lists are very easily accessed via a drop-down at the top of the home page. 
I think the Home/Local/Federated chooser could be ditched, as it’s quite rare that I switch between them like this. I’d find it more useful to have List access there instead.

**Comment by prplecake** [comment]
> I think the Home/Local/Federated chooser could be ditched, as it’s quite rare that I switch between them like this. I’d find it more useful to have List access there instead.

Hard disagree. I switch between timelines often, so I think we should augment this rather than replace.

**Comment by codebykat** [comment]
I don't use the Local or Federated views at all (except when I accidentally swipe to view details of a post, because Tweetbot). Personally I'd like to get rid of them and be able to switch between a list and my full timeline.

**Comment by dgoosens** [comment]
hi

I don't see the point of getting rid of them...  
I personally don't use them either, but understand others do

Best approach, IMHO, is to either replace the current 3 buttons with a drop down where home, local & federated are listed with the user's lists  
OR  
add more buttons, one for every list, and give the user the possibility to slide from one to the other  
OR  
add a dropdown button on the right of the current 3 buttons giving quick and easy access to the user's lists

Either ways, one ought to privilege adding the lists feature... not replacing the existing home/local/federated one

**Comment by Jmuccigr** [comment]
I don't think swiping on a toot should result in a switch to a different timeline, but something specific to that toot (a la TweetBot as others have noted). I'm another person who doesn't switch between timelines very often anyway.

**Comment by ShredderWalnut** [comment]
For what it's worth, here's my personal usage patterns:

- I have a single list and that I wish to stay up-to-date with
- I frequent my Home TL slightly less frequently
- My instance has a pleasant and quiet local TL, and I visit it about twice per day which is enough to allow me to read all the posts
- I never visit the federated TL

**Comment by kerim** [comment]
My thought is to make tapping and holding (long tap) on the "timeline" button pop up a list of lists, and also a setting to allow pinning three items of your own choice to the tabs at the top of the timeline list. For some people this could be home/local/federated, for others it could be home/list1/list2.
